src/flamegraph.cpp

```diff
@@ -447,6 +448,24 @@ bool FlameGraph::eventFilter(QObject* object, QEvent* event)
         updateTooltip();
     } else if (event->type() == QEvent::Hide) {
         setData(nullptr);
+    } else if (event->type() == QEvent::MouseButtonPress) {
```

Comment:
I originally tried to do it all in event->type() == QEvent::ContextMenu but When I tried to get the FrameGraphicsItem I always got NULL. I checked the position returned from mapFromGlobal and it was off by one pixel in x and one pixel in y, so I made the one pixel adjustment and voila! I get our item... not sure what is worse the old way or the new way...

QPoint itemPosition(QCursor::pos().x()-1, QCursor::pos().y()-1);
auto item = static_cast<FrameGraphicsItem*>(m_view->itemAt(m_view->mapFromGlobal(itemPosition)));